requirements/test_redirects.txt

```diff
@@ -0,0 +1,9 @@
+-r dev.txt
```

Comment:
The redirect tests do, as they use `pytest-django` that at least requires `Django` itself. I didn't really want to add `dev.txt` as a requirement for the functional tests as they only currently run against an independently running instance of Bedrock.